models/user_test.go

```diff
@@ -2,10 +2,33 @@ package models
 
 import (
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/require"
 )
 
+func TestIsLocked(t *testing.T) {
+	timestamp := time.Now().Add(-time.Duration(5) * time.Minute)
+	u := &User{
+		LockedAt: &timestamp,
+	}
+	require.False(t, u.IsLocked(0), "user should not be locked")
```

Comment:
personally, I don't add comments to the error unless it is unclear or I am testing something that is convoluted. But that is just personal taste - nothing to take to the bank.